requirements.txt

```diff
@@ -0,0 +1,3 @@
+Cython==0.24.1
```

Comment:
pip didn't like that qcore depended on Cython too and was giving me some error about installing Cython. Let me try that again now that tests are passing.